### ISSUE TITLE
Custom button, input_select entity and Custom buttons again. 

### ISSUE THREAD

**Issue by askusbloodfist** [custom-cards/button-card#310]
**Checklist**
- [X] I updated the card to the latest version available
- [X] I cleared the cache of my browser

**Describe the bug**
When i use the custom button input_select entities i get transparent buttons. 
It happen when i load the page for the first time after restart or if i remove cashe.

If i change views back and forth its as it should be. 
among what i use to make the remote is.

- Vertical / horizontal card. 
- Custom button
- Condition
- Enteties with Input_select *when ever this is in the bug apears* 

Ive tried Other entities and then problem goes away. For example a switch or light.

Ive included screenshots and txt of the code. 
I use the Lovelace webui code editor and allso used the visual basic web from hassio to check if code is wrong or something. 

but after months of testing and trying to fix it i dono what im missing or if its just a bugg. 

**Version of the card**
Version:  latest

**To Reproduce**
[remote.txt](https://github.com/custom-cards/button-card/files/4467714/remote.txt)

**Screenshots**
![4b22cb3c195de369305f10db29a4f8943e6789df_2_230x500](https://user-images.githubusercontent.com/11772079/79087117-a51ff200-7d3e-11ea-8a5e-0a1bcd9f39f1.jpeg)
![ae9eff8c7f3365a145307e4981bbb57341093de3_2_230x500](https://user-images.githubusercontent.com/11772079/79087120-a6511f00-7d3e-11ea-82ad-c28d9e332dde.jpeg)

**Expected behavior**

**Desktop (please complete the following information):**
Dont experiance it in chrome or any pc browser.
Mainly on the Safari app on ios or Homeassistant app. 

**Smartphone (please complete the following information):**
Iphone 6-Iphonexs

**Additional context**
Ive tested this for months now to try to trangulate what causes the problem by removing part after part from my remote app settings. And as i go down the list its when ever i put in a Input_select the problem start. It i put in a switch,light or anything allse on the entities it wont happen.

**Comment by RomRider** [comment]
I don't see how this is related to button-card as it affects other stuff that just button-card.
You probably have one of your top buttons leaking into the view (unclosed html tag or something else).
Does it happen if you remove the 3 top buttons?
Could you share the **whole view** config please?

**Comment by askusbloodfist** [comment]
> I don't see how this is related to button-card as it affects other stuff than just button-card (eg: input_select)
> You probably have one of your top buttons leaking into the view (unclosed html tag or something else).
> Does it happen if you remove the 3 top buttons?
> Could you share the **whole view** config please?

Yes it happens even if i remove the top. Ive tested it with only this card alone and it still happens. 
From my experiance and testing it is something to do with the Button and the input_select cause when i use button row and then input_select and then button row again this will happen. 

Ill post my whole raw config here, And show you a test were everything is removed but the card config i sent already. 

From what i can see its all about when using entities input_select together with the custom button card in a vertical horizontal setup.

![IMG_2882](https://user-images.githubusercontent.com/11772079/79115724-e63bf480-7d86-11ea-8554-82e342a2e4ad.PNG)

[whole config.txt](https://github.com/custom-cards/button-card/files/4469248/whole.config.txt)

**Comment by RomRider** [comment]
I don't know, I don't see anything wrong...
Again this is unrelated to button-card as it affects also the input_select so I won't loose too much time with this.
What you can try is to disable all your custom resources (all the js) and keep just button-card and see what happens on this page (you'll have to reload the resource from the menu also on top right), clear your cache also to be sure.

If the behaviour disappear, one of your custom card is the culprit. Re-enable them 1 by 1 until you see it again and you'll find the problem.

**Comment by askusbloodfist** [comment]
You are right. It was acualy the Compact header causing it. interesting. 

Hm so what in it causes that problem together with the rest. Well one step closer to the problem atleast. yey
